test.js

```diff
@@ -11,8 +11,18 @@ tape('remove accents from string', function(t) {
 	t.end();
 });
 
+tape('remove cyrillic accents from string', function(t) {
+	var input = 'ЁёЙй';
```

Comment:
Ideally, this would go into the large test that covers all supported characters, but not a big deal.